src/components/dashboard/GovernanceSection/GovernanceSection.tsx

```diff
@@ -1,36 +1,50 @@
 import { Typography, Grid, Card, Box } from '@mui/material'
-import { WidgetBody, WidgetContainer } from '@/components/dashboard/styled'
+import { WidgetBody } from '@/components/dashboard/styled'
+import ExpandMoreIcon from '@mui/icons-material/ExpandMore'
+import Accordion from '@mui/material/Accordion'
+import AccordionSummary from '@mui/material/AccordionSummary'
+import AccordionDetails from '@mui/material/AccordionDetails'
+import css from './styles.module.css'
 
 const GovernanceSection = () => (
   <Grid item xs={12} md>
-    <WidgetContainer>
-      <Typography component="h2" variant="subtitle1" fontWeight={700} mb={2}>
-        Governance
-      </Typography>
+    <Accordion className={css.accordion}>
+      <AccordionSummary expandIcon={<ExpandMoreIcon color="border" />}>
+        <div>
```

Comment:
I don't think this is needed.

Reply:
The `<AccordionSummary />` has display: flex so it was placing the titles after each other
<img width="756" alt="Screenshot 2022-11-10 at 17 24 20" src="https://user-images.githubusercontent.com/32431609/201150880-c8444b50-0369-49d0-914a-3b4471a9f17f.png">